src/dsm/headers/Dynamics.hpp

```diff
@@ -286,7 +279,7 @@ namespace dsm {
     void evolve(F f, Tn... args);
 
     /// @brief Get the mean speed of the agents in \f$m/s\f$
-    /// @return Measurement<double> The mean speed of the agents and the standard deviation
```

Comment:
You templated again the struct so this docs need a revert. (I'm only going to pin one of them)